app/models/form_item.rb

```diff
@@ -97,7 +97,7 @@ def as_json(options = {})
     result = super(options)
   end
 
-  def has_group_child?
+  def group_children?
```

Comment:
this seems more ambiguous to me, thinking about other people reading the code. I wouldn't know what group_children means - it could mean are children grouped? should children be grouped?